presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -2361,6 +2361,14 @@ public void testRightJoinWithNullValues()
     }
 
     @Test
+    public void testJoinWithDuplicateRelations()
+            throws Exception
+    {
+        assertQuery("SELECT * FROM orders JOIN orders USING (orderkey)");
+        assertQuery("SELECT * FROM lineitem x JOIN orders x USING (orderkey)");
```

Comment:
Nevermind, I realized that `assertQuery` also takes `expectedSql` which is executed by h2 runner. So, I added equivalent queries that are also valid for h2.